.github/workflows/testing/ci-daily-local.yml

```diff
@@ -22,6 +22,9 @@ jobs:
     - name: Install package dependencies / prepare workspaces
       run: yarn install --frozen-lockfile
 
+    - name: Testing Prettier format

```

Comment:
This was previously left out of code coverage due to another problem. Now that this PR fixes coverage issue, I'm also adding this back in to have consistency with other pipelines.